index.js

```diff
@@ -5,6 +5,6 @@ module.exports = {
   name: 'ember-country-region-selector',
 
   included: function(app) {
-    app.import('bower_components/country-region-selector/dist/crs.min.js');
+    app.import(app.bowerDirectory + '/country-region-selector/dist/crs.js');
```

Comment:
One other thing I did was to change it from pulling in the minified file to the fat version.  `ember build` minifies all dependencies anyway, so we can get away with not doing it here.  This helps in development, if somebody needs to work with breakpoints and such